alignmentpro/alignmentapp/api.py

```diff
@@ -374,3 +383,36 @@ def list(self, request):
                 "results": serializer.data,
             }
         )
+
+
+@api_view(['GET', 'POST'])
+@authentication_classes((TokenAuthentication, SessionAuthentication,))
+@permission_classes((IsAuthenticated,))
+def review_section(request):
+    if request.method == 'POST':
+        data = json.loads(request.body)
+        section = DocumentSection.objects.get(pk=data['section_id'])
+        section.text = data['section_text']
+        section.reviewed_by = request.user
+        # TODO: Add a 'Finalize' button to the UI, so that if they need to stop, they can
+        # save and come back to finish later. We may also want to only allow finalizing after a review.
+        if 'finalize' in data and data['finalize']:
+            section.is_draft = False
+            # TODO: Add UserAction points for this.
+        section.save()
+
+    else:
+        section = DocumentSection.get_section_for_review()
+        if not section:
+            return Response({'error': 'No document sections currently available for review. Please check back again later.'})
+    image_url = "{}scans/{}/{}".format(settings.MEDIA_URL, section.get_section_dir(), section.name + '_chunk001_lowres.png')
```

Comment:
Note that certain topics have multiple chunks `_chunk001`, `_chunk002`, ... (at most three chunks)
so need to load all of them here, and response should be more `image_urls` instead of `image_url`